src/CMakeLists.txt

```
@@ -6,108 +6,99 @@ add_library(tuvx_object OBJECT)
 
 target_compile_features(tuvx_object PUBLIC cxx_std_11)
 
-set_target_properties(tuvx_object PROPERTIES
-  Fortran_MODULE_DIRECTORY ${TUVX_MOD_DIR}
-)
+set_target_properties(tuvx_object PROPERTIES Fortran_MODULE_DIRECTORY
+                                             ${TUVX_MOD_DIR})
 
-target_include_directories(tuvx_object
-  PUBLIC
-    $<BUILD_INTERFACE:${TUVX_MOD_DIR}>
-    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include>
-    $<INSTALL_INTERFACE:${TUVX_INSTALL_MOD_DIR}>
-    $<INSTALL_INTERFACE:${TUVX_INSTALL_INCLUDE_DIR}>
-)
+target_include_directories(
+  tuvx_object
+  PUBLIC $<BUILD_INTERFACE:${TUVX_MOD_DIR}>
+         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include>
+         $<INSTALL_INTERFACE:${TUVX_INSTALL_MOD_DIR}>
+         $<INSTALL_INTERFACE:${TUVX_INSTALL_INCLUDE_DIR}>)
 
 message(STATUS "blas libraries: ${BLAS_LIBRARIES}")
 message(STATUS "lapack libraries: ${LAPACK_LIBRARIES}")
 
-target_link_libraries(tuvx_object 
-  PRIVATE
-    PkgConfig::netcdff
-    PkgConfig::netcdfc
-    yaml-cpp::yaml-cpp
-)
+target_link_libraries(tuvx_object PRIVATE PkgConfig::netcdff PkgConfig::netcdfc
+                                          yaml-cpp::yaml-cpp)
```

Can this formatting be reverted or be made consistent with similar changes above?
```suggestion
target_link_libraries(tuvx_object PRIVATE PkgConfig::netcdff
                                          PkgConfig::netcdfc
                                          yaml-cpp::yaml-cpp)
```